package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "sw3",
+  "name": "sw3_with_ERC20",
```

Comment:
I don't know if we should change the name. I don't think so, but I did it because I wanted to work on it on my own fork.